app/controllers/maps_controller.rb

```diff
@@ -177,7 +177,10 @@ def region
     area = params[:id] || "this area"
     @title = "Maps in #{area}"
     ids = Map.bbox(params[:minlat],params[:minlon],params[:maxlat],params[:maxlon]).collect(&:id)
-    @maps = Map.where(password: '').where('id IN (?)',ids).paginate(:page => params[:page], :per_page => 24)
+    @maps = Map.where(password: '').where('id IN (?)',ids).paginate(:page => params[:page], :per_page => 24).except(:styles)
+    @maps.each do |map|
+      map[:image_urls] = map.warpables.map{ |warpable| warpable.image.url}
```

Comment:
Wow cool! Does this return an array? 

What do you think of writing a functional test for this, and we can confirm the output? 

Thanks!!! This is awesome!